src/components/Sidebar.jsx

```diff
@@ -113,6 +117,9 @@ function Sidebar(props) {
                         />
                     </div>
                 </div>
+                <div className="container d-flex flex-column align-items-center font-barlow fw-medium p-0 pb-3">
+                    <span className="px-3 py-1 rounded text-white bg-steel-blue">{user.username}</span>
+                </div>
```

Comment:
```suggestion
                <div className="container d-flex flex-column align-items-center font-barlow fw-medium p-0 pb-3">
                    {user.role !== 'GUEST' && user.role !== 'ADMIN' ? (
                        <Link
                            className="px-3 py-1 rounded text-decoration-none text-white bg-steel-blue"
                            to={isDashboard ? '/dash/profile' : '/profile'}
                            onClick={() => closeSidebar()}
                        >
                            {user.username}
                        </Link>
                    ) : (
                        <span className="px-3 py-1 rounded text-white bg-steel-blue">{user.username}</span>
                    )}
                </div>
```
- Apenas sugestão. Geralmente clicar no seu username em interfaces tende a levar para seu perfil. Coincidiu que este elemento já parece um botão;